public/legacy/modules/Emails/include/ListView/FoldersViewModal.js

```diff
@@ -87,7 +87,9 @@
             var mbox = data.selected[0];
             // reload with different inbox
             $('[name=folders_id]').val(mbox);
-            top.location = 'index.php?module=Emails&action=index&folders_id=' + mbox;
+            //Fixed #117 - Disappearing Main Menu for Users With Multiple Inboxes
+            // Suitecrm 8 Modify the iframe link not the main query
+            location = 'index.php?module=Emails&action=index&folders_id=' + mbox;
```

Comment:
Hi @abuzarfaris,

Once again thank you for your contributions.

You don't need to add the comment with id of the issue. We prefer to not have it and add that info to the commit message body instead.

So could you please remove the following comment and add that info to the commit, please (if you think its worth)

```
            //Fixed #117 - Disappearing Main Menu for Users With Multiple Inboxes
            // Suitecrm 8 Modify the iframe link not the main query
```

Reply:
Removed the comment